### PR TITLE
feat: add more space to headers so content is more easily scannable

### PR DESCRIPTION
**Pull request by danawoodman**
When scanning the large API docs I find it difficult to scan the content. This change adds a bit more vertical spacing between sections which I believe will help people scan content more easily and distinguish sections more clearly. Also made the headings slightly larger.

### CODE REVIEW

File: public/styles/_base.scss [hunk 1]
```diff
@@ -84,10 +84,10 @@ img[data-src] {
 // add top margin
 // to make it easier to see headers within a long wall of text
 .strong-headers {
-  h1 {                      font-weight: 300; -webkit-font-smoothing: initial; }
-  h2 { margin-top: 2em;     font-weight: 300; -webkit-font-smoothing: initial; border-bottom: $border; }
-  h3 { margin-top: 1.75em;  font-weight: 300; -webkit-font-smoothing: initial; }
-  h4 { margin-top: 1.5em;   font-weight: 300; }
+  h1 {                      	  		 font-weight: 300; -webkit-font-smoothing: initial; }
+  h2 { margin: 2em 0 1em;   	font-size: 32px; font-weight: 300; -webkit-font-smoothing: initial; border-bottom: $border; }
```
Comment (HashimotoYT) [inline public/styles/_base.scss]:
```suggestion
  h2 { margin: 1.75em 0 1em;   	font-size: 32px; font-weight: 300; -webkit-font-smoothing: initial; border-bottom: $border; }
```

File: public/styles/_base.scss [hunk 2]
```diff
@@ -84,10 +84,10 @@ img[data-src] {
...
+  h3 { margin: 2.25em 0 0.75em; font-size: 26px; font-weight: 300; -webkit-font-smoothing: initial; }
```
Comment (HashimotoYT) [inline public/styles/_base.scss]:
```suggestion
  h3 { margin: 2em 0 0.75em; font-size: 26px; font-weight: 300; -webkit-font-smoothing: initial; }
```

**Comment by danawoodman** [comment]
I’ve accepted the changes. I think a general increase in spacing is good for legibility. Specific sizes were just suggestions 👍

**Comment by danawoodman** [comment]
Woo! Thanks!

**Pull request merged**